test/unit/cmap/connection.test.ts

```diff
@@ -72,7 +73,7 @@ class FakeSocket extends EventEmitter {
   }
 }
 
-class InputStream extends Readable {
+class InputStream extends Duplex {
```

Comment:
For my own understanding, why was this changed?

Reply:
Malik and I changed this debugging, but now this is unnecessary.  We should revert any changes that aren't necessary for this PR